### ISSUE TITLE
a11y_NativeSelect> Has Error Inverse> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#505]
**Describe the bug**
NativeSelect>> "1 violations" is appearing for "Has Error Inverse". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/nativeselect--has-error-inverse'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for NativeSelect> Has Error Inverse.

**Screenshots**
https://somup.com/c3n2QnZW9h

**Desktop:**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
The issue is still Reproducible.
See Screen-cast : http://somup.com/c3jffluCp0
Thanks!

**Comment by ccedrone** [comment]
This appears to be an issue with Storybook and not the code itself. Leslie Schulz verified it as a11y compliant. No plans to fix.
